src/index.js

```diff
@@ -26,13 +17,11 @@ export { default as CardContent } from "./Card/CardContent";
 export { default as CardSection } from "./Card/CardSection";
 
 export { default as Heading } from "./Heading";
-export { default as Text } from "./Text/Text";
+export { default as Text } from "./Text";
 export { default as Icon } from "./Icon";
 export { default as InputField } from "./InputField";
 export { default as TextLink } from "./TextLink";
-export { default as ThemeProvider } from "./Theming/ThemeProvider";
 export { default as Radio } from "./Radio";
 export { default as Select } from "./Select";
 export { default as Textarea } from "./Textarea";
-export { withTheme } from "theming";
 export { defaultTokens as defaultTheme, getTokens } from "@kiwicom/orbit-design-tokens";
```

Comment:
Yes, because we can't export nested object because it shouldn't be usable

bad example:

```
<ThemeProvider theme={{ custom: customTheme, orbit: defaultTokens }} />
```
in this case default tokens is nested to `orbit` so the result is

```
orbit: {
 orbit: defaultTokens
}
```